modules/cco-ccoctl-creating-individually.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc
+
+[id="cco-ccoctl-creating-individually_{context}"]
+= Creating AWS resources individually
+
+// Would be nice to have a little explanation why a user would want to go this route vs. the other option.
```

Comment:
The details in this section is basically pulling back the curtain and showing how this all really works under the covers. I think the main use case here is that the individual/company wants to delegate responsibility for creating the various objects across different company departments.